### DISCUSSION TITLE
Lesson 6 virtual machine platfrom download

### DISCUSSION THREAD

**Discussion by leojawad19** [Cyfrin/foundry-full-course-cu#1113] [General]
How do i install virtual machine platfrom on windows whenever i try to install it says "Catastrophic failure". 
What should i do? Can anyone help me out. 
[
![25](https://github.com/Cyfrin/foundry-full-course-f23/assets/137924034/82ac7599-0537-401d-984c-7b69d4b918f6)
](url)

**Comment by 0xusmanf** [comment]
Can you please explain how you are trying to install it and after which step you get this error? are you following the guidelines explained in the tutorial for Windows?

**Reply by leojawad19** [reply]
Yes and i am pretty sure the video only covered the linux part at 6:35:20 (0 - 6)

**Reply by 0xusmanf** [reply]
Make sure you are logged on to Windows from an admin account and then run the power shell as an administrator.

Then Install.

**Reply by leojawad19** [reply]
Same thing, how do i know if i am running the power shell as an administrator.

**Reply by 0xusmanf** [reply]
Right-click on the power shell icon and click on run as administrator.

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/110072406/27f84c36-d599-4349-b536-5f5371801de0)

**Reply by leojawad19** [reply]
Still 
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/137924034/ea06537b-6d5e-403b-ac97-6182acb5b09c)

**Reply by 0xusmanf** [reply]
Do you have multiple user profiles on your PC?

**Reply by leojawad19** [reply]
I only have 1 rn but it's a custom window

**Reply by 0xusmanf** [reply]
In the last snap when you ran the power shell as an administrator, I noticed your path changed to.
```shell
C:\Windows\system32>
``` 
which was originally 
```shell
C:\Users\Hammad>
```
 Make sure your path stays the same as `C:\Users\Hammad>` while you run the power shell as administrator and then try.

**Reply by leojawad19** [reply]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/137924034/c11bf3f4-a682-42b0-bec9-9061af5873e0)

It's different when i run as administrator.

**Reply by 0xusmanf** [reply]
Do you have an administrator written with your user profile name like in the screenshot below?

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/110072406/dd728fe1-432e-40ef-bb2b-4d9565653a5b)

**Reply by leojawad19** [reply]
Yes

**Reply by 0xusmanf** [reply]
After running the power shell as an administrator run below command
```shell
cd ..
```
Then again
```shell
cd ..
```
Then
```shell
cd Users\Hammad
```
Then try installing.

**Reply by leojawad19** [reply]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/137924034/c3b2298e-c43c-4e7e-96f2-e5461b486d82)

**Reply by leojawad19** [reply]
Not working.

**Reply by 0xusmanf** [reply]
Have you tried installing wsl after following the above instructions?

As per the screenshot above you are trying to install wsl before following the above instructions.

**Reply by 0xusmanf** [reply]
One more thing you can check. Please ensure that virtualization is enabled in your computer's BIOS.

**Reply by leojawad19** [reply]
Sorry for bothering you again
Now it says this 
![30](https://github.com/Cyfrin/foundry-full-course-f23/assets/137924034/35a40630-3b22-4b99-a6f7-5e356ca81e37)

**Reply by 0xusmanf** [reply]
> One more thing you can check. Please ensure that virtualization is enabled in your computer's BIOS.

Enable this.

**Reply by leojawad19** [reply]
It is enabled. 
![31](https://github.com/Cyfrin/foundry-full-course-f23/assets/137924034/aaebd898-63c8-4c26-8992-3953c642c9a6)

**Reply by 0xusmanf** [reply]
Make sure this is also on

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/110072406/b99066f1-d893-42f3-a3db-1d808bfaf5c7)

**Reply by leojawad19** [reply]
Virtual Machine platform is enabled and if i reboot my system how would that work as shown in the video at 6:35:36 (0 - 6)

**Reply by 0xusmanf** [reply]
You need to reboot the system after successfully installing WSL, which you haven't yet, so I don't think rebooting would do any good.

Edit: I see, your last snap shows you have installed WSL and you got the error after your system rebooted. In this case try running the `wsl` command.

**Reply by leojawad19** [reply]
![33](https://github.com/Cyfrin/foundry-full-course-f23/assets/137924034/a3f153e5-4b0f-47b6-8d96-8946f37b0382)

**Reply by leojawad19** [reply]
@usmanfarooq91 Ty for your time and effort.

**Reply by 0xusmanf** [reply]
Have you had any luck installing it?

**Reply by leojawad19** [reply]
Despite their efforts to help me on Patrick's dicord server, the problem persists

**Reply by 0xusmanf** [reply]
Just found this [thread](https://github.com/microsoft/WSL/discussions/8364) on this issue, it might help you.
